pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -985,6 +989,13 @@ func Run(ps *pubsub.PubSub) {
 			if fatalFlag {
 				log.Fatal("Requested fault injection to cause watchdog")
 			}
+
+		case <-zedagentCtx.getCertsTimer.C:
+			start := time.Now()
+			ok := getCloudCertChain(&zedagentCtx)
+			log.Infof("zedagent Run: getCertsTimer get cloud cert ok %v\n", ok)
+			pubsub.CheckMaxTimeTopic(agentName, "getCertsTimer", start,
+				warningTime, errorTime)
```

Comment:
Better to keep this block above the stillRunning code.

Reply:
ok.